src/urls.ts

```diff
@@ -26,3 +26,8 @@ export function stringifyQueryString(query: NextRouter["query"]): string {
   });
   return params.toString();
 }
+export function objectifyQueryString(query: string): NextRouter["query"] | undefined {
```

Comment:
This always returns a value, so:
```suggestion
export function objectifyQueryString(query: string): NextRouter["query"] {
```

Reply:
`UrlObject` type has `query` declared as optional